README.md

```diff
@@ -4,13 +4,18 @@
 
 ## To generate / update `*pb.go` files
 
-* Install `protoc-gen-go`, 
+* Install `protoc-gen-go`:
+```
+go install google.golang.org/protobuf/cmd/protoc-gen-go@latest
+```
 
-* run `go mod vendor` in `ccloud-sdk-go-v1-public`
+* Run `go mod vendor`
 
-* at `ccloud-sdk-go-v1-public` root dir, run the following command:
+* At `ccloud-sdk-go-v1-public` root dir, run the following command, replace `org/org.go` with relevant file
```

Comment:
I think the command above also implies that we're in the root directory, so it should be fine to leave this out.
```suggestion
* Run the following command, replacing `org/org.go` with the relevant file
```